src/bun.js/bindings/BunProcess.cpp

```diff
@@ -2871,6 +3005,7 @@ extern "C" void Process__emitDisconnectEvent(Zig::GlobalObject* global)
   binding                          Process_functionBinding                             Function 1
   browser                          constructBrowser                                    PropertyCallback
   chdir                            Process_functionChdir                               Function 1
+  channel                          constructProcessChannel                             PropertyCallback
```

Comment:
Should this be `DontEnum`?

Reply:
it could be. i dont feel strongly in either direction